game/ui/base/transactionscreen.cpp

```diff
@@ -386,18 +386,27 @@ void TransactionScreen::populateControlsAlien()
 {
 	int leftIndex = getLeftIndex();
 	int rightIndex = getRightIndex();
+
 	for (auto &ae : state->agent_equipment)
 	{
+		auto &alienTypeName = ae.first;
```

Comment:
Can be const?

Reply:
Updated to const:

https://github.com/OpenApoc/OpenApoc/blob/12f960b12bf8b7d40fd104f027fedb298ccdcd30/game/ui/base/transactionscreen.cpp#L390-L392